webapp/src/components/App/Singlestudy/explore/Modelization/BindingConstraints/index.tsx

```diff
@@ -73,25 +74,25 @@ function BindingConstraints() {
 
   return (
     <UsePromiseCond
-      response={constraints}
-      ifPending={() => <SimpleLoader />}
+      response={constraintsRes}
+      ifPending={() => <Skeleton sx={{ height: 1, transform: "none" }} />}
```

Comment:
I think it's a bad idea to use Skeleton like this, it makes the loading not clear...
For me we keep the spinner until to use Skeleton correctly.

Reply:
Same message for all others Skeleton